lib/takeaway.rb

```diff
@@ -0,0 +1,45 @@
+require 'twilio-ruby'
+require_relative 'menu'
+require_relative 'text'
+
+
+class TakeAway
+
+  include Text
+
+  attr_reader :menu, :order, :order_total
+
+  def initialize
+    @menu = Menu.new
+    @order = []
+    @order_total = 0
+  end
+
+  def read_menu
+    @menu.menu_list.each { |key, value| puts "#{key}, £#{value}"}
+  end
+
+  def add_item_to_order(menu_item, quantity=1)
+    quantity.times { |order| @order << menu_item}
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `order`. You can omit the argument if you don't care about it. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)